src/main/scala/com/typesafe/sbt/site/SitePreviewPlugin.scala

```diff
@@ -34,11 +41,23 @@ object SitePreviewPlugin extends AutoPlugin {
       server stop()
       server destroy()
     },
+    previewAuto := {
+      val port = previewFixedPort.value getOrElse Port.any
+      val sLog = streams.value.log
+      val browser = previewLaunchBrowser.value
+
+      Preview(port, (target in previewAuto).value, makeSite, watchSources, state.value) run { server =>
+        if(browser)
+          Browser open(server.portBindings.head.url)
+      }
+      sLog.info("SitePreviewPlugin server started on port %d. Press any key to exit." format port)
```

Comment:
This is too late, it will only print after the server has been started.

Also it seems something else is printing as well:

```
Embedded server listening at
  http://127.0.0.1:4000
Press any key to stop.

[info] SitePreviewPlugin server started on port 4000. Press any key to exit.
```